validator/lib/validatePnts.js

```diff
@@ -196,6 +198,42 @@ function validatePnts(content) {
         }
     }
 
+    if (defined(featureTableJson.RGBA)) {
+        featureTable.featuresLength = pointsLength * 4;
+        componentDatatype = ComponentDatatype.fromName(defaultValue(featureTableJson.RGBA.componentType, 'UNSIGNED_SHORT', 4));
```

Comment:
I have purposely set the parameter to `UNSIGNED_SHORT`. When I set it to `UNSIGNED_BYTE`, the values, if out of range, will be clipped when creating an array using `featureTable.getPropertyArray()`. So I am using higher precision type to catch invalid values. Same for the other checks.